### PR TITLE
Settings-waitinglistPositionRework

### PR DESCRIPTION
**Pull request by PrinzPizza42**

### CODE REVIEW

File: src/main/java/org/example/DeathMinigames/commands/GameCMD.java [hunk 1]
```diff
@@ -100,6 +100,14 @@ public void execute(CommandSourceStack stack, String[] args) {
                     player.sendMessage(Component.text(tf.getTranslation(player, "diffAt")).color(NamedTextColor.GOLD)
                             .append(Component.text(config.checkConfigInt(player, "Difficulty")).color(NamedTextColor.RED)));
                     break;
+                case "setwaitinglistposition":
```
Comment (JonasFranke) [inline src/main/java/org/example/DeathMinigames/commands/GameCMD.java:103]: cases should be added to tab complete

File: src/main/java/org/example/DeathMinigames/commands/GameCMD.java [hunk 2]
```diff
@@ -100,6 +100,14 @@ public void execute(CommandSourceStack stack, String[] args) {
...
+                    Location location = player.getLocation();
+                    config.setWaitingListPosition(location);
+                    player.sendMessage(Component.text(tf.getTranslation(player, "setWaitingListPosition")).color(NamedTextColor.GREEN));
+                    break;
+                case "decidednottosetposition":
```
Comment (JonasFranke) [inline src/main/java/org/example/DeathMinigames/commands/GameCMD.java:108]: same here

**Review by JonasFranke — APPROVED** [review]
lgtm. sry I couldn't review it earlier

File: src/main/java/org/example/DeathMinigames/commands/GameCMD.java (hunk 1 above)
Comment (PrinzPizza42) [reply src/main/java/org/example/DeathMinigames/commands/GameCMD.java:103]: I actually did that fully aware that these commands wont be shown to the player. I only want them to be executed by using methods I provided, like a clickevent in chat or the settings menu.

Comment (JonasFranke) [reply src/main/java/org/example/DeathMinigames/commands/GameCMD.java:103]: k

**Pull request merged**